Linux/Build/Deployment/azure-pipelines.yml

```diff
@@ -0,0 +1,101 @@
+trigger:
+  batch: true
+  branches:
+    include:
+      - master
+  paths:
+    include:
+      - "Linux/Build/Deployment"
+    exclude:
+      - "*"
+
+pr:
+  paths:
+    include:
+      - "Linux/Build/Deployment"
+
+resources:
+  repositories:
+  - repository: das-platform-building-blocks
+    type: github
+    name: SkillsFundingAgency/das-platform-building-blocks
+    ref: refs/heads/DASD-9551_Verify_K8_Manifest_Apply
+    endpoint: SFA
+  pipelines:
+  - pipeline: build
+    source: das-azure-pipelines-agents-linux-build (Image Build)
+    branch: DASD-9551_Test_Agents
+    trigger:
+      branches:
+        - master
+
+pool:
+  vmImage: ubuntu-latest
+
+variables:
+- group: RELEASE das-azure-pipelines-agent
+- group: Release Management Resources
+- name: ContainerImageName
+  value: azure-pipelines-build-agent-ubuntu-20
+- name: ImageTag
+  value: $(resources.pipeline.build.runName)
```

Comment:
This is a  bit unusual and the docs are hard to find, took me a while to figure it out.  Might be worth linking to the docs in a comment - https://docs.microsoft.com/en-us/azure/devops/pipelines/process/resources?view=azure-devops&tabs=schema#pipeline-resource-variables

Might also be worth using a less generic name for the pipeline resource alias, maybe `image-build` instead of `build` so the variable would look like `$(resources.pipeline.image-build.runName)`

Reply:
Thanks it works making it image-build again! I started with image-build and took it out because It was in inline powershell (which I changed) and escaping the - wasn't going well.